invenio_rdm_records/marshmallow/json.py

```diff
@@ -36,34 +42,235 @@ class PersonIdsSchemaV1(StrictKeysMixin):
+            raise ValidationError(
+                _('Invalid title type. Not one of {allowed}.'.format(
+                                                    allowed=self.TITLE_TYPES)),
+                field_names=['additional_titles']
+            )
+
+
+class DescriptionSchemaV1(StrictKeysMixin):
+    """Schema for the additional descriptions."""
+
+    DESCRIPTION_TYPES = [
+          "Abstract",
+          "Methods",
+          "SeriesInformation",
+          "TableOfContents",
+          "TechnicalInfo",
+          "Other"
+    ]
+    description = SanitizedUnicode(required=True,
+                                   validate=validate.Length(min=3))
+    description_type = SanitizedUnicode()
+    lang = SanitizedUnicode()
+
+    @validates('lang')
+    def validate_language(self, value):
+        """Validate that language is ISO 639-3 value."""
+        validate_iso639_3(value)
+
+    @validates('description_type')
+    def validate_description_type(self, value):
+        """Validate that the description type is one of the allowed ones."""
+        if value not in self.DESCRIPTION_TYPES:
+            raise ValidationError(
+                _('Invalid description type. Not one of {allowed}.'.format(
+                                            allowed=self.DESCRIPTION_TYPES)),
+                field_names=['additional_descriptions']
+            )
+
+
+class DateSchemaV1(StrictKeysMixin):
+    """Schema for date intervals."""
+
+    DATE_TYPES = [
+        "Collected",
+        "Valid",
+        "Withdrawn"
+    ]
+
+    start = DateString()
+    end = DateString()
+    type = fields.Str(required=True)
+    description = fields.Str()
+
+    @validates('type')
+    def validate_type(self, value):
+        """Validate that the type is one of the allowed ones."""
+        if value not in self.DATE_TYPES:
+            raise ValidationError(
+                _('Invalid date type. Not one of {allowed}.'.format(
+                                            allowed=self.DATE_TYPES)),
+                field_names=['dates']
+            )
+
+
+class RightSchemaV1(StrictKeysMixin):
+    """Schema for rights."""
+
+    rights = SanitizedUnicode()
+    uri = SanitizedUnicode()
+    identifier = SanitizedUnicode()
+    identifier_scheme = SanitizedUnicode()
+    scheme_uri = SanitizedUnicode()
+    lang = SanitizedUnicode()
+
+    @validates('lang')
+    def validate_language(self, value):
+        """Validate that language is ISO 639-3 value."""
+        validate_iso639_3(value)
+
 
 class MetadataSchemaV1(StrictKeysMixin):
     """Schema for the record metadata."""
 
     # TODO: Check enumeration (i.e. only open/embargoed/... accepted)
     access_right = SanitizedUnicode(required=True)
     access = fields.Nested(AccessSchemaV1)
-    recid = PersistentIdentifier()
-    title = SanitizedUnicode(required=True, validate=validate.Length(min=3))
+    additional_descriptions = fields.List(fields.Nested(DescriptionSchemaV1))
```

Comment:
Would that be enough for what we need:

```suggestion
    additional_descriptions = fields.Nested(DescriptionSchemaV1, many=True)
```

Reply:
I think it is. However, I saw this issue https://github.com/marshmallow-code/marshmallow/issues/779 and I am quite reluctant to do any big change now until we fully migrate to Marshmallow 3 and that RFC gets accepted/rejected.

What do you think?